cli/src/main.rs

```diff
@@ -184,7 +189,22 @@ fn main() {
     }
 }
 
-fn get_file_paths(input_patterns: Vec<String>) -> Result<Vec<PathBuf>, GlobError> {
+fn get_file_paths(input_patterns: Vec<String>, exclude_patterns: Vec<String>) -> Result<Vec<PathBuf>, GlobError> {
+    let exclude_patterns = exclude_patterns
+        .into_iter()
+        .filter_map(|exclude_pattern| {
```

Comment:
Instead of silently ignoring / filtering incorrect glob patterns, I would expect it to throw an error, to avoid accidentally formatting files that it was not supposed to. In the suggested, I collect to a result of `Result<Vec<Pattern>, PatternError>` and panic if it is incorrect (not the best way of error handling, but I will overhaul this later on for the CLI crate as a whole)